conductr_cli/sandbox_stop.py

```diff
@@ -1,8 +1,16 @@
-from conductr_cli import sandbox_stop_docker
+from conductr_cli import sandbox_stop_docker, sandbox_stop_jvm
+from conductr_cli.sandbox_common import major_version, RUNNING_CONDUCTR_VERSION_FILE_PATH
 
 
 def stop(args):
     """`sandbox stop` command"""
 
-    sandbox_stop_docker.stop(args)
+    if not hasattr(args, 'image_version'):
+        with open(RUNNING_CONDUCTR_VERSION_FILE_PATH, 'r') as file:
+            args.image_version = file.read()
+
+    if args.image_version == 1:
+        sandbox_stop_docker.stop(args)
+    elif args.image_version != 1:
+        sandbox_stop_jvm.stop(args)
```

Comment:
I think we should call stop for both docker and jvm based sandbox, since I think the user would expect both sandbox to stop if they are both running.